src/Nethermind/Nethermind.Synchronization/Peers/PeerClientType.cs

```diff
@@ -1,27 +1,27 @@
 //  Copyright (c) 2018 Demerzel Solutions Limited
 //  This file is part of the Nethermind library.
-// 
+//
 //  The Nethermind library is free software: you can redistribute it and/or modify
 //  it under the terms of the GNU Lesser General Public License as published by
 //  the Free Software Foundation, either version 3 of the License, or
 //  (at your option) any later version.
-// 
+//
 //  The Nethermind library is distributed in the hope that it will be useful,
 //  but WITHOUT ANY WARRANTY; without even the implied warranty of
 //  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 //  GNU Lesser General Public License for more details.
-// 
+//
 //  You should have received a copy of the GNU Lesser General Public License
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 
 namespace Nethermind.Synchronization.Peers
 {
     public enum PeerClientType
     {
-        BeSu,
-        Geth,
-        Nethermind,
-        Parity,
-        Unknown
+        BeSu = 0,
+        Geth = 1,
+        Nethermind = 2,
+        Parity = 3,
+        Unknown = 255
```

Comment:
why 255? also 0,1,2,3 are assigned automatically
we should maybe move Unknown to front so it is a default value.

anyway why would you change these in minimal changes?